html/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta http-equiv="Refresh" content="0; url='https://github.com/scpwiki/interwiki'" />
+</head>
+
+</html>
```

Comment:
Missing newline at end of file
```suggestion
</html>
```

Reply:
Force-pushed a reformat with Prettier (my editor was formatting with some other default)